tests_react_native/package.json

```diff
@@ -13,7 +13,10 @@
     "cross-env": "^7.0.2",
     "prop-types": "^15.8.1",
     "react": "^17.0.2",
-    "react-native": "0.67.3"
+    "react-dom": "^17.0.2",
+    "react-native": "0.67.3",
+    "react-native-web": "^0.17.7",
+    "react-scripts": "^5.0.0"
```

Comment:
Suggest a react-scripts dependency that is wrapped so you can do config overrides
I use to use craco (we still do in our authentication example and the luna template...) but it is in flux at the moment:
https://github.com/gsoft-inc/craco/issues/415

Looks like react-app-rewired might be the way to go
https://github.com/timarney/react-app-rewired/pull/530#issuecomment-1001025219

until/unless they make extensibility a part of create-react-app: https://github.com/facebook/create-react-app/issues/12362